### ISSUE TITLE
Crash report "java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class com.github.adamantcheese.chan.ui.view.TouchBlockingLinearLayout"

### ISSUE THREAD

**Issue by K1rakishou** [Adamantcheese/Kuroba#878]
Crash report
UUID: a548b33089b352bdd4cf11604a4fa15b
Build flavor: dev
Version name: v4.12.0-db0938f
OS info: Android 9, sdk version: 28
Report description: No title
Logs:
```
=== LOGS(2020-05-07 18:40:51 UTC) ===
E/Kuroba-dev | UNCAUGHT: Called from unhandled exception handler.
E/Kuroba-dev | UNCAUGHT: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class com.github.adamantcheese.chan.ui.view.TouchBlockingLinearLayout
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3430)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:112)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:216)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:7625)
E/Kuroba-dev | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
E/Kuroba-dev | UNCAUGHT: Caused by: android.view.InflateException: Binary XML file line #27: Binary XML file line #27: Error inflating class com.github.adamantcheese.chan.ui.view.TouchBlockingLinearLayout
E/Kuroba-dev | UNCAUGHT: Caused by: android.view.InflateException: Binary XML file line #27: Error inflating class com.github.adamantcheese.chan.ui.view.TouchBlockingLinearLayout
E/Kuroba-dev | UNCAUGHT: Caused by: java.lang.reflect.InvocationTargetException
E/Kuroba-dev | UNCAUGHT: 	at java.lang.reflect.Constructor.newInstance0(Native Method)
E/Kuroba-dev | UNCAUGHT: 	at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.createView(LayoutInflater.java:658)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:801)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.rInflate(LayoutInflater.java:874)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:835)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
E/Kuroba-dev | UNCAUGHT: 	at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
E/Kuroba-dev | UNCAUGHT: 	at com.github.adamantcheese.chan.utils.LayoutUtils.inflate(LayoutUtils.java:22)
E/Kuroba-dev | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.controller.DrawerController.onCreate(DrawerController.java:94)
E/Kuroba-dev | UNCAUGHT: 	at com.github.adamantcheese.chan.StartActivity.onCreate(StartActivity.java:135)
E/Kuroba-dev | UNCAUGHT: 	at android.app.Activity.performCreate(Activity.java:7458)
E/Kuroba-dev | UNCAUGHT: 	at android.app.Activity.performCreate(Activity.java:7448)
E/Kuroba-dev | UNCAUGHT: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1286)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3409)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
E/Kuroba-dev | UNCAUGHT: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:112)
E/Kuroba-dev | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:216)
E/Kuroba-dev | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:7625)
E/Kuroba-dev | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
E/Kuroba-dev | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
E/Kuroba-dev | UNCAUGHT: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040038 a=-1}
E/Kuroba-dev | UNCAUGHT: 	at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:1001)
E/Kuroba-dev | UNCAUGHT: 	at android.content.res.TypedArray.getDrawable(TypedArray.java:985)
E/Kuroba-dev | UNCAUGHT: 	at android.view.View.<init>(View.java:5050)
E/Kuroba-dev | UNCAUGHT: 	at android.view.ViewGroup.<init>(ViewGroup.java:659)
E/Kuroba-dev | UNCAUGHT: 	at android.widget.LinearLay
E/Kuroba-dev | UNCAUGHT: ------------------------------
E/Kuroba-dev | UNCAUGHT: END OF CURRENT RUNTIME MESSAGES
E/Kuroba-dev | UNCAUGHT: ------------------------------
E/Kuroba-dev | UNCAUGHT: Android API Level: 28
E/Kuroba-dev | UNCAUGHT: App Version: v4.12.0-3c28f30
D/Kuroba-dev | AndroidUtils: Build Signature: 10d4e433
E/Kuroba-dev | UNCAUGHT: Development Build: Development
E/Kuroba-dev | UNCAUGHT: Phone Model: HUAWEI CLT-L29
... TRUNCATED ...

=== SETTINGS ===
Prefetching enabled: false
Thread downloading enabled: false, active downloads = 0
Hi-res thumbnails enabled: true
Youtube titles parsing enabled: true
Youtube durations parsing enabled: false
Concurrent file loading chunks count: 2
WEBM streaming enabled: false
Saved files base dir info: Java API location: /storage/emulated/0/Kuroba-dev/files, SAF location: content://com.android.externalstorage.documents/tree/primary%3AKuroba, active: SAF
Local threads base dir info: Java API location: /storage/emulated/0/Kuroba-dev/saved_threads, SAF location: , active: Java API
Phone layout mode: AUTO
OkHttp IPv6 support enabled: true
OkHttp HTTP/2 support enabled: true
```

**Comment by Adamantcheese** [comment]
This appears to be a crash because backcolor (0x7f040038 in attr in resources.arsc; toss in a debug APK to Android Studio to browse through the contents) isn't resolving to a drawable, which I guess is expected since it is a color attribute as specified in attrs,xml. However it works for some devices and I haven't touched any part of that specific resource attribute with any recent changes, despite there being a lot of them. Not exactly sure what's going on there.

**Comment by Adamantcheese** [comment]
I know the cause, but I can't reproduce it. I've added reference tags to the color attributes, but I don't know if that will help at all.

**Comment by Adamantcheese** [comment]
It's an attempt, at least with this it should be trying to get a color instead of a drawable; still have no idea how to test it though.

**Comment by lewri** [comment]
Logcat from latest build (excuse my potential overgiving of data).

```
05-18 10:29:34.955 13605 13616 E         : [ZeroHung]zrhung_get_config: Get config failed for wp[0x0008]
05-18 10:29:34.956 13605 13624 E AwareLog: AtomicFileUtils: readFileLines file not exist: android.util.AtomicFile@41d2e09
05-18 10:29:34.957 13605 13624 E AwareLog: AtomicFileUtils: readFileLines file not exist: android.util.AtomicFile@869880e
05-18 10:29:35.052 13605 13616 E         : [ZeroHung]zrhung_get_config: Get config failed for wp[0x0008]
05-18 10:29:35.193 13605 13605 W cheese.chan.de: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setUseSessionTickets(Z)V (light greylist, reflection)
05-18 10:29:35.193 13605 13605 W cheese.chan.de: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setHostname(Ljava/lang/String;)V (light greylist, reflection)
05-18 10:29:35.193 13605 13605 W cheese.chan.de: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->getAlpnSelectedProtocol()[B (light greylist, reflection)
05-18 10:29:35.193 13605 13605 W cheese.chan.de: Accessing hidden method Lcom/android/org/conscrypt/OpenSSLSocketImpl;->setAlpnProtocols([B)V (light greylist, reflection)
05-18 10:29:35.199 13605 13605 W cheese.chan.de: Accessing hidden method Ldalvik/system/CloseGuard;->get()Ldalvik/system/CloseGuard; (light greylist, reflection)
05-18 10:29:35.199 13605 13605 W cheese.chan.de: Accessing hidden method Ldalvik/system/CloseGuard;->open(Ljava/lang/String;)V (light greylist, reflection)
05-18 10:29:35.199 13605 13605 W cheese.chan.de: Accessing hidden method Ldalvik/system/CloseGuard;->warnIfOpen()V (light greylist, reflection)
05-18 10:29:35.280 13605 13621 E MemoryLeakMonitorManager: MemoryLeakMonitor.jar is not exist!
05-18 10:29:35.286 13605 13634 E         : [ZeroHung]zrhung_get_config: Get config failed for wp[0x0102]
05-18 10:29:35.286 13605 13634 W ZrHung.AppEyeUiProbe: Failed to get config from zrhung
05-18 10:29:35.293 13605 13616 E         : [ZeroHung]zrhung_get_config: Get config failed for wp[0x0008]
05-18 10:29:35.341 13605 13605 W cheese.chan.de: Accessing hidden method Landroid/graphics/drawable/Drawable;->getOpticalInsets()Landroid/graphics/Insets; (light greylist, linking)
05-18 10:29:35.341 13605 13605 W cheese.chan.de: Accessing hidden field Landroid/graphics/Insets;->left:I (light greylist, linking)
05-18 10:29:35.341 13605 13605 W cheese.chan.de: Accessing hidden field Landroid/graphics/Insets;->right:I (light greylist, linking)
05-18 10:29:35.341 13605 13605 W cheese.chan.de: Accessing hidden field Landroid/graphics/Insets;->top:I (light greylist, linking)
05-18 10:29:35.341 13605 13605 W cheese.chan.de: Accessing hidden field Landroid/graphics/Insets;->bottom:I (light greylist, linking)
--------- beginning of crash
05-18 10:29:35.482 13605 13605 E AndroidRuntime: FATAL EXCEPTION: main
05-18 10:29:35.482 13605 13605 E AndroidRuntime: Process: com.github.adamantcheese.chan.dev, PID: 13605
05-18 10:29:35.482 13605 13605 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: android.view.InflateException: Binary XML file line #17: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3430)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:112)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:216)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7625)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
05-18 10:29:35.482 13605 13605 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #17: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.482 13605 13605 E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.482 13605 13605 E AndroidRuntime: Caused by: java.lang.reflect.InvocationTargetException
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at java.lang.reflect.Constructor.newInstance0(Native Method)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.createView(LayoutInflater.java:658)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:801)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.utils.LayoutUtils.inflate(LayoutUtils.java:22)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.ui.controller.StyledToolbarNavigationController.onCreate(StyledToolbarNavigationController.java:40)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.controller.Controller.addChildController(Controller.java:146)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.ui.controller.DrawerController.setChildController(DrawerController.java:129)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.StartActivity.setupLayout(StartActivity.java:290)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.StartActivity.onCreate(StartActivity.java:139)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7458)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7448)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1286)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3409)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:112)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:216)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7625)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
05-18 10:29:35.482 13605 13605 E AndroidRuntime: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040038 a=-1}
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:1001)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.content.res.TypedArray.getDrawable(TypedArray.java:985)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.View.<init>(View.java:5054)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.view.ViewGroup.<init>(ViewGroup.java:659)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.widget.FrameLayout.<init>(FrameLayout.java:92)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.widget.FrameLayout.<init>(FrameLayout.java:87)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at android.widget.FrameLayout.<init>(FrameLayout.java:82)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        at com.github.adamantcheese.chan.ui.view.TouchBlockingFrameLayout.<init>(TouchBlockingFrameLayout.java:33)
05-18 10:29:35.482 13605 13605 E AndroidRuntime:        ... 29 more
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Called from unhandled exception handler.
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.github.adamantcheese.chan.dev/com.github.adamantcheese.chan.StartActivity}: android.view.InflateException: Binary XML file line #17: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3430)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.os.Handler.dispatchMessage(Handler.java:112)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.os.Looper.loop(Looper.java:216)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.main(ActivityThread.java:7625)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at java.lang.reflect.Method.invoke(Native Method)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Caused by: android.view.InflateException: Binary XML file line #17: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Caused by: android.view.InflateException: Binary XML file line #17: Error inflating class <unknown>
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Caused by: java.lang.reflect.InvocationTargetException
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at java.lang.reflect.Constructor.newInstance0(Native Method)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.createView(LayoutInflater.java:658)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:801)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:741)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.inflate(LayoutInflater.java:492)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.utils.LayoutUtils.inflate(LayoutUtils.java:22)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.ui.controller.StyledToolbarNavigationController.onCreate(StyledToolbarNavigationController.java:40)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.controller.Controller.addChildController(Controller.java:146)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.ui.controller.DrawerController.setChildController(DrawerController.java:129)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.StartActivity.setupLayout(StartActivity.java:290)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.github.adamantcheese.chan.StartActivity.onCreate(StartActivity.java:139)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.Activity.performCreate(Activity.java:7458)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.Activity.performCreate(Activity.java:7448)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1286)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3409)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3614)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:86)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2199)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.os.Handler.dispatchMessage(Handler.java:112)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.os.Looper.loop(Looper.java:216)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.app.ActivityThread.main(ActivityThread.java:7625)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at java.lang.reflect.Method.invoke(Native Method)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Caused by: java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f040038 a=-1}
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.content.res.TypedArray.getDrawableForDensity(TypedArray.java:1001)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.content.res.TypedArray.getDrawable(TypedArray.java:985)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.View.<init>(View.java:5054)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at android.view.ViewGroup.<init>(ViewGroup.java:659)
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT:         at a
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: ------------------------------
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: END OF CURRENT RUNTIME MESSAGES
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: ------------------------------
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: Android API Level: 28
05-18 10:29:35.483 13605 13605 E Kuroba-dev | UNCAUGHT: App Version: v4.12.0-35d2530
05-18 10:29:35.484 13605 13605 E Kuroba-dev | UNCAUGHT: Development Build: Development
05-18 10:29:35.484 13605 13605 E Kuroba-dev | UNCAUGHT: Phone Model: HUAWEI STF-L09
05-18 10:29:35.519 13605 13616 E         : [ZeroHung]zrhung_get_config: Get config failed for wp[0x0008]
05-18 10:29:35.746 13605 13605 W cheese.chan.dev: type=1400 audit(0.0:737244): avc: granted { create } for name="crashlog_1589794175484.txt" scontext=u:r:untrusted_app:s0:c15,c257,c512,c768 tcontext=u:object_r:sdcardfs:s0:c15,c257,c512,c768 tclass=file
```

**Comment by Adamantcheese** [comment]
@lewri, I've modified some things and everything looks good on my end, but hopefully these recent changes should fix this issue. If you can test it out for me that would be nice.

**Comment by lewri** [comment]
Yep working fine now 👍

**Comment by Adamantcheese** [comment]
Closed as of 40ce675, greatly appreciated for the help.
